Doc/edits based on rendered doc (#715)
pyansys/pyfluent: doc/source/api/core/solver/index.rst

Solver
======

Th``solver`` module allows you to use Fluent solver capabilities from Python. You
can use both :ref:`ref_solver_tui`-based commands and :ref:`ref_settings`. You allows

> have access to Fluent field data (surface, scalar, and vector), events, and monitoring. 

Reviewer: Please review wording.

> Th``solver`` module allows you to use Fluent solver capabilities from Python. You
> can use both :ref:`ref_solver_tui`-based commands and :ref:`ref_settings`. You allows
> have access to Fluent field data (surface, scalar, and vector), events, and monitoring. 

Author: ```suggestion
The ``solver`` module allows you to use Fluent solver capabilities from Python. You
can use both :ref:`ref_solver_tui` commands and :ref:`ref_settings` to access Fluent
events, monitoring, and field data (surface, scalar, and vector). 
```
